### PR TITLE
Fix i2ctool

### PR DESCRIPTION
**Pull request by tadziopazur**
Fix build break triggered by enabling SYSTEM_I2CTOOL
Integrate i2c command into tash

### CODE REVIEW

File: apps/shell/tash_command.c [hunk 1]
```diff
@@ -94,6 +94,10 @@ extern tash_taskinfo_t tash_taskinfo_list[];
 static int is_sorted_list = FALSE;
 static struct tash_cmd_info_s tash_cmds_info = {PTHREAD_MUTEX_INITIALIZER};
 
+#ifdef CONFIG_SYSTEM_I2CTOOL
```
Comment (sunghan-chang) [inline apps/shell/tash_command.c]: Let's use builtin functionality. Please refer [README](https://github.com/Samsung/TizenRT/blob/master/apps/builtin/README.md) of builtin.

File: os/arch/arm/src/s5j/s5j_i2c.c [hunk 2]
```diff
@@ -1331,6 +1331,10 @@ struct i2c_dev_s *up_i2cinitialize(int port)
 	return (FAR struct i2c_dev_s *)priv;
 }
 
+int up_i2cuninitialize(FAR struct i2c_dev_s *dev) {
```
Comment (sunghan-chang) [inline os/arch/arm/src/s5j/s5j_i2c.c]: This is a different kind of change. Let's split commit.

File: apps/shell/tash_command.c (hunk 1 above)
Comment (tadziopazur) [reply apps/shell/tash_command.c]: Will do.

Comment (sunghan-chang) [reply apps/shell/tash_command.c]: Thank you.

**Comment by tadziopazur** [comment]
Split into https://github.com/Samsung/TizenRT/pull/547(merged) and https://github.com/Samsung/TizenRT/pull/550 (not yet ready).

**Pull request closed without merging**